spec/spec_helper_acceptance.rb

```diff
@@ -1,192 +1,82 @@
-require 'mustache'
-require 'open3'
-require 'rbvmomi'
-require 'retries'
-
-# This exception is used to signal expected continuations when waiting for events on the vCenter
-class NotFinished < RuntimeError
-end
-
-class PuppetManifest < Mustache
-  def initialize(file, config)
-    @template_file = File.join(Dir.getwd, 'spec', 'acceptance', 'fixtures', file)
-    config.each do |key, value|
-      config_value = self.class.to_generalized_data(value)
-      instance_variable_set("@#{key}".to_sym, config_value)
-      self.class.send(:attr_accessor, key)
-    end
-  end
-
-  def apply
-    manifest = render.delete("\n")
-    cmd = "bundle exec puppet apply --detailed-exitcodes -e \"#{manifest}\" --modulepath spec/fixtures/modules --libdir lib --debug"
-    result = { output: [], exit_status: nil }
-
-    Open3.popen2e(cmd) do |_stdin, stdout_err, wait_thr|
-      while line = stdout_err.gets
-        result[:output].push(line)
-        puts line
-      end
-      result[:exit_status] = wait_thr.value
-    end
-
-    result
+# frozen_string_literal: true
```

Comment:
I _think_ we should use the litmus `configure!` method for our spec_helper_acceptance files now:

```ruby
require 'puppet_litmus'
require 'spec_helper_acceptance_local' if File.file?(File.join(File.dirname(__FILE__), 'spec_helper_acceptance_local.rb'))

PuppetLitmus.configure!
```